game-core/src/main/java/games/strategy/triplea/delegate/power/calculator/AvailableSupportCalculator.java

```diff
@@ -61,7 +48,7 @@ public static SupportCalculationResult getSortedSupport(
    * @param defence are the receiving units defending?
    * @param allies are the receiving units allied to the giving units?
    */
-  private static SupportCalculationResult getSupport(
+  public static SupportCalculationResult getSupport(
```

Comment:
Codacy found an issue: [The method 'getSupport(Collection, Set, boolean, boolean)' has an NPath complexity of 3603](https://app.codacy.com/gh/triplea-game/triplea/pullRequest?prid=6317773)